app/views/shared/_links.haml

```diff
@@ -1,8 +1,9 @@
-%li= link_to 'diasporafoundation.org', "https://diasporafoundation.org"
-%li= link_to 'Wiki', "https://wiki.diasporafoundation.org"
-%li= link_to t('layouts.application.whats_new'), changelog_url
-%li= link_to t('layouts.header.code') + " " + pod_version, "#{source_url}", {:title => t('layouts.application.source_package')}
+%li= link_to "diasporafoundation.org", "https://diasporafoundation.org"
+%li= link_to "Wiki", "https://wiki.diasporafoundation.org"
+%li= link_to "Forum", "https://discourse.diasporafoundation.org"
+%li= link_to t("layouts.application.whats_new"), changelog_url
+%li= link_to t("layouts.header.code") + " " + pod_version, "#{source_url}", {:title => t("layouts.application.source_package")}
```

Comment:
- Line is too long. [127/120]
- Prefer `to_s` over string interpolation.
- Redundant curly braces around a hash parameter.
- Use the new Ruby 1.9 hash syntax.